app/assets/stylesheets/application.css.scss

```diff
@@ -90,7 +90,11 @@ a, a:visited, a:link {
 }
 
 .flash {
-    text-align: center;
+  text-align: center;
```

Comment:
Properties should be ordered -moz-box-shadow, -webkit-box-shadow, box-shadow, font-size, text-align